pkg/util/k8sutil/pods.go

```diff
@@ -536,19 +548,20 @@ type ArangodbExporterContainerConf struct {
 	JWTTokenSecretName string
 	LivenessProbe      *HTTPProbeConfig
 	Image              string
+	Resources          v1.ResourceRequirements
 }
 
 // CreateArangodPod creates a Pod that runs `arangod`.
 // If the pod already exists, nil is returned.
 // If another error occurs, that error is returned.
 func CreateArangodPod(kubecli kubernetes.Interface, developmentMode bool, deployment APIObject,
-	role, id, podName, pvcName, image, lifecycleImage, alpineImage string,
-	imagePullPolicy v1.PullPolicy, imagePullSecrets []string,
-	engine string, requireUUID bool, terminationGracePeriod time.Duration,
-	args []string, env map[string]EnvValue, finalizers []string,
-	livenessProbe *HTTPProbeConfig, readinessProbe *HTTPProbeConfig, tolerations []v1.Toleration, serviceAccountName string,
-	tlsKeyfileSecretName, rocksdbEncryptionSecretName string, clusterJWTSecretName string, nodeSelector map[string]string,
-	podPriorityClassName string, resources v1.ResourceRequirements, exporter *ArangodbExporterContainerConf, sidecars []v1.Container, vct *v1.PersistentVolumeClaim) error {
```

Comment:
I am waiting for the day that someone cleans this up.

Reply:
https://github.com/arangodb/kube-arangodb/pull/495#issue-336725977

Reply:
Test created. 
Pods are rotated. 
I have created a new task for cleaning up.